miio/miot_device.py

```diff
@@ -27,7 +27,7 @@ def get_properties_for_mapping(self) -> list:
         properties = [{"did": k, **v} for k, v in self.mapping.items()]
 
         return self.get_properties(
-            properties, property_getter="get_properties", max_properties=15
+            properties, property_getter="get_properties", max_properties=10
```

Comment:
I had to reduce max_properties to 10 or else i do get "Checksum error" from my device.